Gemfile

```diff
@@ -49,6 +49,10 @@ group :development, :test do
   gem "debug", platforms: %i[ mri windows ]
 
   gem 'faker'
+
+  gem 'mailcatcher', '~> 0.2.4'
+
+  gem 'thin', '~> 1.8', '>= 1.8.2'
```

Comment:
On top of this add a comment. something like this:
`This gem is required with mailcatcher as without this mailcatcher shows <XYZ-error>. The solution was suggestion on https://www.example.com `

This is just a format, fill out details correctly for this.

Following this process gives clear indication why we need this and also helps developers in future to be aware of side-effects because of this gem.

Reply:
Provided comments about the error and why separate installation is required

Reply:
Looks good. Thanks.